sys/include/net/unicoap.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2024-2025 Carl Seifert
+ * Copyright (C) 2024-2025 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+#ifndef NET_UNICOAP_H
+#define NET_UNICOAP_H
+
+#include "net/unicoap/constants.h"
+#include "net/unicoap/config.h"
+#include "net/unicoap/message.h"
+#include "net/unicoap/options.h"
```

Comment:
```suggestion
#include "net/unicoap/constants.h" /* IWYU pragma: export */
#include "net/unicoap/config.h" /* IWYU pragma: export */
#include "net/unicoap/message.h" /* IWYU pragma: export */
#include "net/unicoap/options.h" /* IWYU pragma: export */
```

Since the point of this header is to make the other headers available, we can pass this information on to the static analyzer with the IWYU pragmas. That way those will not suggest to rather include `net/unicoap/constants.h` directly, instead of `net/unicoap.h`.